src/basic_async_model/basic_server.cxx

```diff
@@ -0,0 +1,102 @@
+#include "../basic_async_model/basic_server.hxx"
```

Comment:
target_include_directories in cmake is a better alternative it will exclude a need to use the relative include path

Reply:
Accorded to leave it for later reconsideration.